pylhc/BPM_calibration.py

```diff
@@ -0,0 +1,593 @@
+from pathlib import Path
```

Comment:
Todo:
Add docstring header like in the other entrypoint files.
You can use the create param help from generic parser
```
from generic_parser.entrypoint_parser import create_parameter_help

from omc3.utils.logging_tools import get_logger

LOG = get_logger(__name__, fmt='(message)s')

from pylhc import forced_da_analysis as module
create_parameter_help(module)
```

You also need to add things to the `doc/entrypoints/` as `.rst` (see there)

Reply:
.. this should go into the wiki...